### PR TITLE
Updates Outreach Committee section

### PR DESCRIPTION
**Pull request by pdp2shirts**
Updates the Outreach Committee section of the SUBGROUPS.md
Signed-off-by: Paul Pindell <ppindell@gmail.com>

### CODE REVIEW

**Comment by pdp2shirts** [comment]
@opiproject/opi-maintainers It doesn't appear that I have rights to request a review from specific people. So using this backdoor method to request a review :)

File: SUBGROUPS.md
Comment (seroyer) [inline SUBGROUPS.md]: A few nits:
1. Prefer to wrap long lines
2. "sometimes" -> "Sometimes"
3. Second sentence is incomplete

**Review by seroyer — APPROVED** [review]

File: SUBGROUPS.md
Comment (pdp2shirts) [reply SUBGROUPS.md]: @seroyer fixed 1, 2, and 3.

File: SUBGROUPS.md
Comment (mestery) [inline SUBGROUPS.md]: Can you move these sentences to line up with `The` above? I'm somewhat surprised this passed the linter.

Comment (pdp2shirts) [reply SUBGROUPS.md]: done

**Review by glimchb — APPROVED** [review]
approved and merging this now

**Pull request merged**
